packages/amplify-cli/src/execution-manager.ts

```diff
@@ -120,6 +131,13 @@ async function selectPluginForExecution(context: Context, pluginCandidates: Plug
       pluginCandidates = consoleHostingPlugins.concat(otherPlugins);
     }
 
+    const amplifyMeta = context.amplify.getProjectMeta();
+    const { Region } = amplifyMeta.providers['awscloudformation'];
+
+    if (!isContainersEnabled(context) || Region !== 'us-east-1') {
```

Comment:
Whats special ablout `us-east-1`

Reply:
CloudFront only supports ACM certificates in the US East (N. Virginia) Region (us-east-1).

https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_ViewerCertificate.html